src/webpack/index.ts

```diff
@@ -75,3 +84,55 @@ function removeMiniCssExtract(config: any) {
     }
   });
 }
+
+function getLibraryName(options: any) {
+  if (options?.customWebpackConfig?.libraryName) {
+    return options.customWebpackConfig.libraryName;
+  }
+
+  const projectName = getProjectNameFromAngularJson();
+  if (projectName) return projectName;
+
+  console.warn(
+    'Warning: single-spa-angular could not determine a library name to use and has used a default value.',
+  );
+  console.info('This may cause issues if this app uses code-splitting or lazy loading.');
+  if (!options) {
+    console.info('You may also need to update extra-webpack.config.json.');
+  }
+  console.info('See <SomeUrl> for information on how to resolve this.');
```

Comment:
I think that linking to an explanation would work better than trying to include it in the console output.
Of course, this explanation still needs to be written, and the url determined.

Reply:
Perhaps we could add some documentation to the Angular page at https://github.com/single-spa/single-spa.js.org/blob/master/website/versioned_docs/version-5.x/ecosystem-angular.md?

Reply:
Agreed. That seemed like the most reasonable place to put it.

Reply:
I was thinking a new section before "Use Angular Builder" under manual installation could briefly document manually setting up the custom-webpack builder and document the new options, and we could point the users to that section.

Reply:
I think the Use Angular Builder section of the docs refers to the Angular 7 builder, not to the custom webpack builder. I think that this could be near the section about what the schematics do?